### PR TITLE
issue #153 : model changes

### PR DESCRIPTION
**Pull request by adisos**

### CODE REVIEW

File: pkg/ibmvpc/vpc.go [hunk 1]
```diff
@@ -125,6 +134,14 @@ type VPC struct {
...
 	nodes             []vpcmodel.Node
 	connectivityRules *vpcmodel.ConnectivityResult // allowed connectivity between elements within the vpc
+	zones             map[string]*Zone
+}
+
+func (v *VPC) getZoneByName(name string) (*Zone, error) {
+	if z, ok := v.zones[name]; ok {
+		return z, nil
+	}
+	return nil, errors.New("zone not found")
```
Comment (zivnevo) [inline pkg/ibmvpc/vpc.go:144]: Might be helpful to add zone name and vpc name to the error message

**Review by haim-kermany — DISMISSED** [review]

File: pkg/ibmvpc/parser.go [hunk 2]
```diff
@@ -465,20 +502,41 @@ func getFipConfig(
...
+			nodes:       []vpcmodel.Node{},
+			zones:       map[string]*Zone{},
+		}
 		res.NodeSets = append(res.NodeSets, vpcNodeSet)
 		res.NameToResource[vpcNodeSet.Name()] = vpcNodeSet
 	}
 }
 
+func singleVPCErr() error {
+	return errors.New("expecting single vpc")
```
Comment (zivnevo) [inline pkg/ibmvpc/parser.go:516]: This error message is a bit misleading. Should be "NACL/SG VPC (<vpc1>) is different from its layer's VPC (<vpc2>)."

**Review by zivnevo — APPROVED** [review]
Just these two tiny error-handling issues. Otherwise, looks good.

File: pkg/ibmvpc/vpc.go (hunk 1 above)
Comment (adisos) [reply pkg/ibmvpc/vpc.go]: done

File: pkg/ibmvpc/parser.go (hunk 2 above)
Comment (adisos) [reply pkg/ibmvpc/parser.go]: done

**Review by zivnevo — APPROVED** [review]

**Pull request merged**